variables.tf

```diff
@@ -193,6 +193,12 @@ variable "nessus_activation_codes" {
   default     = []
 }
 
+variable "nessus_web_server_port" {
+  type        = number
+  description = "The port on which the Nessus web server should listen (e.g. 8834)."
+  default     = 8834
+}
+
```

Comment:
We _could_ verify that the number is within the expected range (`[1, 65535]`).  This is just a thought and need not impede the approval of this PR.

Reply:
Arg- looks like `strcontains()` wasn't added until Terraform 1.5, so it is [failing in our current linter](https://github.com/cisagov/cool-assessment-terraform/actions/runs/5871426698/job/15920738947?pr=222#step:15:91) because we are still on [Terraform 1.2.3](https://github.com/cisagov/setup-env-github-action/blob/e40f45df94a2084271806c3651c7ffe177a9d9ef/action.yml#L43-L44).

I will fix this, but probably not until tomorrow.

Reply:
Is there any reason not to update [cisagov/setup-env-github-action](https://github.com/cisagov/setup-env-github-action)?  I'm rockin' 1.5.4 locally over here.

Reply:
Not that I'm aware of.  I am going to make a PR shortly in that repo.

Reply:
https://github.com/cisagov/setup-env-github-action/pull/66 has been created to address this.  I will resolve this conversation after that PR has been merged and I verify that this branch passes linting.

Reply:
https://github.com/cisagov/setup-env-github-action/pull/66 has been merged and the tests in this PR are now passing. @jsf9k are you good with resolving this conversation?

Reply:
Yep.